src/store/slices/drive/index.ts

```diff
@@ -242,6 +243,7 @@ const downloadFileThunk = createAsyncThunk<
 
       trackDownloadSuccess();
     } catch (err) {
+      logger.error('downloadFileThunk ', JSON.stringify(err));
```

Comment:
Maybe it would be better to add a clearer message like `Error downloading a file` or `Error in downloadFileThunk` or something like that.